src/utils/websocket.js

```diff
@@ -13,11 +15,15 @@ function constructWebsocketURL(type, period) {
 
 export function wsChanges(dispatch, type, period) {
   let url = constructWebsocketURL(type, period)
-  const ws = new WebSocket(url)
-  ws.onmessage = function (msg) {
+  const options = {
+    connectionTimeout: 10000,
+    maxRetries: 10,
```

Comment:
Can we remove this line? The default is `Infinity` with geometric backoff, which is what we want.

This is my test:
1. Null route websocket traffic (however you want to do it, deleting manger is fine too)
2. Refresh UI page
3. Wait 101 seconds
4. Restore websocket connectivity

Reply:
Yeah, that sounds good.

@LLParse can you fork this PR and update it? Remember give credit to @ly5156 .